five_tips_about_testable_golang_code.md

```diff
@@ -0,0 +1,171 @@
+# How to write UNtestable code in golang
+According to my experience in movies, if you want to be a super bad guy, you will need to work harder than the good guys. Think about it! You need to hire evil scientists, build a huge death bomb to threaten the world, a good plan and a lot of money. But be a good guy you just basically need some brave hearts, you do not even have to know about the structure of bomb because when choosing red/blue wire to cut, good guys never mess up.
+
+Yeah, that's not fair.
+
+Good news (?) everyone, in software developer side, mess up code is far easier than build it in a good way. If you keep following points in mind, make golang code untestable would be so easy.
```

Comment:
-messing up code is far easier than building it in a good way

-making golang code untestable would be so easy